operator/src/main/java/org/keycloak/operator/controllers/WatchedSecretsController.java

```diff
@@ -49,16 +49,14 @@
 import java.util.stream.Collectors;
 
 import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.inject.Inject;
 
 import static io.javaoperatorsdk.operator.api.reconciler.Constants.WATCH_CURRENT_NAMESPACE;
 
 @ApplicationScoped
 @ControllerConfiguration(namespaces = WATCH_CURRENT_NAMESPACE, labelSelector = Constants.KEYCLOAK_COMPONENT_LABEL + "=" + WatchedSecrets.WATCHED_SECRETS_LABEL_VALUE)
 public class WatchedSecretsController implements Reconciler<Secret>, EventSourceInitializer<Secret>, WatchedSecrets {
 
-    @Inject
-    KubernetesClient client;
+    private volatile KubernetesClient client;
```

Comment:
Just out of curiosity, can you please elaborate why to make this change?

Reply:
Relates to https://github.com/quarkiverse/quarkus-operator-sdk/issues/702 - the client injected by Quarkus when running locally against openshift doesn't work by default, but the one we create in BaseOperatorTest does which is picked up by this change.